userWhitelist.json

```diff
@@ -22,10 +22,12 @@
     "aks681"
   ],
   "teamLeads": {
-    "onboardingTeam": "Showtim3",
+    "onboardingTeam": "DubeySandeep",
     "releaseTeam": "ankita240796",
```

Comment:
This is now me and Nithesh but I guess you can add just me for now.

Reply:
Done.

Reply:
Uhmm this means that oppiabot will ping you for good first issues since we are currently pinging the onboarding team lead.

Reply:
I meant to replace Ankita with me, and keep DubeySandeep.